package.json

```diff
@@ -64,5 +64,5 @@
     "typescript": "^5.2.2",
     "yaml": "^2.3.4"
   },
-  "packageManager": "yarn@1.22.19"
+  "packageManager": "yarn@4.0.1"
```

Comment:
@ignatiusreza why?

Reply:
> https://yarnpkg.com/blog

Bigger, Better, Faster.. :tm:

Reply:
Let's refactor everything to use Bun 🙈

Reply:
Feel free to give it a try if you want.. I'm not well informed on how compatible Bun is..